site/app/templates/forum/displayThreadList.twig

```diff
@@ -0,0 +1,36 @@
+{% for thread in thread_content %}
+    <a href="{{ thread.link }}">
+        <div class="{{ thread.class }}">
+            {% if thread.pinned == true %}
+                <i class="fas fa-star" style="padding-left:3px;position:relative; float:right; display:inline-block; color:gold; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title = "Announcement" aria-hidden="true"></i>
+            {% endif %}
+            {% if thread.favorite == true %}
+                <i class="fas fa-thumbtack" style="padding-left:3px;position:relative; float:right; display:inline-block; color:gold; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title="Pinned as my favorite" aria-hidden="true"></i>
+            {% endif %}
+            {% if thread.merged_thread_id != -1 %}
+                <i class="fas fa-link" style="padding-left:3px;position:relative; float:right; display:inline-block; color: white; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title="Thread Merged" aria-hidden="true"></i>
+            {% endif %}
+
+            {% if thread.status != 0 %}
+                <i class="fa {{ thread.fa_icon }}" style="margin-right:{{ thread.fa_margin_right }}; padding-left:3px; position:relative; float:right; display:inline-block; color:{{ thread.fa_color }}; font-size:{{ thread.fa_font_size }};" title="{{ thread.tooltip }}" aria-hidden="true"></i>
+            {% endif %}
+
+            {% if thread.is_locked == true %}
+                <i class="fas fa-lock" style="padding-left:3px;position:relative; float:right; display:inline-block; color: white; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title="Locked" aria-hidden="true"></i>
+            {% endif %}
+
+            <h4>{{ thread.title | raw  }}</h4>
```

Comment:
As @MasterOdin mentioned, `| raw` shouldn't be used here (just pass the unfiltered value and twig will take care of this)

Reply:
tried that, still renders as plaintext html!

Reply:
removed the raw by bringing the logic that was adding html to the Twig